A76-ring-hash-improvements.md

```diff
@@ -93,13 +93,22 @@ At pick time:
 then the request hash key will be set to this value. If `request_hash_header`
```

Comment:
For clarity, suggest changing "If request_hash_header" to "If the header".  That way, it's clear that we're talking about the header specified by the `request_hash_header` config field, not the `request_hash_header` config field itself.

Reply:
Done.